etc/profile-m-z/youtube.profile

```diff
@@ -18,7 +18,10 @@ whitelist ${HOME}/.config/Youtube
 
 private-bin electron,electron[0-9],electron[0-9][0-9],youtube
 private-etc @tls-ca,@x11,bumblebee,host.conf,mime.types
-private-opt Youtube
+# private-opt breaks file-copy-limit, use a whitelist instead of draining RAM
+# https://github.com/netblue30/firejail/discussions/5307
+#private-opt Youtube
+whitelist /opt/Youtube
```

Comment:
```suggestion
```

I think it would be better to put this information in firejail(1) instead of
repeating it everywhere for `private-opt` (the docs can be left for later; I
could suggest or add them later).

Suggestion: Delete `private-opt` and the comment and move the `whitelist`
command to the usual place.

Reply:
The suggested changes look very reasonable and shouldn't take long to implement. Seeing your misc. remarks, I'll freeze this for a while so we can have a proper discussion on the private-opt topic in general.

Reply:
@kmk3 Did some extra research on the size of the non-electron apps. I implemented your suggestions, including a warning note to firejail(1). Kept private-opt for enpass (~85MB), mate-dictionary (<20MB), minecraft-launcher (~1.6MB) and ppsspp (~44MB). The only app I couldn't check: xmr-stak (FTBFS on my Arch Linux box). I returned it to the functional state it had prior to this PR.

HTH